extensions/Webhooks/template/en/default/pages/webhooks.html.tmpl

```diff
@@ -0,0 +1,190 @@
+[%# This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
This should probably be moved into /docs as an `.rst` file.

Reply:
Not a bad idea but given the time frame let's file it as a post MVP enhancement to be fixed later.